src/local/butler/deploy.py

```diff
@@ -424,7 +424,7 @@ def _prod_deployment_helper(config_dir,
 
 def execute(args):
   """Deploy Clusterfuzz to Appengine."""
-  if sys.version_info.major != 3 or sys.version_info.minor != 7:
+  if sys.version_info.major != 3:
```

Comment:
Why remove the .7 check? I think things will fail without .7?

Reply:
In my workflow, I use Ubuntu 20.04 to setup the environment, and in `src/local/install_deps_linux.bash` it will check python installed or not. In Ubuntu 20.04, it has installed python3.8 by default, so `python3.8` would be exported as the environment variable. Therefore, when running to `src/local/butler/deploy.py`, the version check will fail.
I am not sure if it is the normal workflow.